packages/protocol/contracts/common/Proxy.sol

```diff
@@ -32,8 +33,17 @@ contract Proxy {
   function () external payable {
     bytes32 implementationPosition = IMPLEMENTATION_POSITION;
 
+    address implementationAddress;
+
+    assembly {
+      implementationAddress := sload(implementationPosition)
+    }
+
+	if (implementationAddress == address(0)) return;
```

Comment:
Can you add a comment here explaining why we do this?

Reply:
:+1: